cartography/intel/aws/iam.py

```diff
@@ -925,6 +929,57 @@ def sync_user_access_keys(
     )
 
 
+def _set_used_state_tx(
+    tx: neo4j.Transaction, project_id: str, common_job_parameters: Dict, update_tag: int,
+) -> None:
+    ingest_role_used = """
+    MATCH (n:AWSRole)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}})
+    <-[:OWNER]-(:CloudanixWorkspace{id: {WORKSPACE_ID}})
+    WHERE (n)-[:TRUSTS_AWS_PRINCIPAL]->() AND n.lastupdated = {update_tag}
```

Comment:
please point me to the code where we determine TRUSTS_AWS_PRINCIPAL relationship

Reply:
Please can you explain little bit it? i can’t understand what you want to say